libs/s3/iam_core_lib.py

```diff
@@ -80,6 +80,7 @@ def __init__(
                                                    endpoint_url=endpoint_url)
             else:
                 LOGGER.info("Skipped: create iam client, resource object with boto3.")
+        # pylint: disable=broad-except
```

Comment:
Rather than disabling the Codacy warning let's narrow down scope of the except block by specifying the botocore.exceptions.ClientError exception.